src/interface_adapter/vote_player/VotePlayerController.java

```diff
@@ -0,0 +1,4 @@
+package interface_adapter.vote_player;
```

Comment:
Name the use case VoteOut instead of VotePlayer so that we know it is not a voting system but instead a way to vote someone out during daytime (kill someone during the day as opposed to werewolves killing a villager at night)